atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -336,6 +336,9 @@ def get_odcs_config(self):
             odcs_config = ODCSConfig(**odcs_config_kwargs)
         return odcs_config
 
+    def is_empty(self):
+        return self.conf != self.DEFAULT_CONFIG
+
```

Comment:
Since the only caller of is_empty() is get_cluster_info() and it negates the return, could this be changed to is_default() and be 'return self.config == self.DEFAULT_CONFIG', and then get_cluster_info would have 'if self.reactor_config.is_default():'

The double negation is confusing.

Reply:
ok

Reply:
actually the logic was wrong there has to be 1 negation, because we wanna set override only if it is NOT default :)